docs/guide/bundles.md

```diff
@@ -24,59 +24,98 @@ To learn more about organizing your eZ Platform project, see [Best Practices](be
 
-Some of the key built-in bundles are:
-
-#### eZ Platform kernel
-
-[ezpublish-kernel](https://github.com/ezsystems/ezpublish-kernel) contains the core of the whole
-eZ Platform application.
-
-##### EzPublishCoreBundle
-
-EzPublishCoreBundle is contained in [ezpublish-kernel](https://github.com/ezsystems/ezpublish-kernel).
-
-To get an overview of EzPublishCoreBundle's configuration, run the following command-line script:
-
-``` bash
-php app/console config:dump-reference ezpublish
-```
-
-#### PlatformUIBundle
-
-[PlatformUIBundle](https://github.com/ezsystems/PlatformUIBundle) provides the main editing and back-end interface for eZ Platform.
-
-#### ezplatform-solr-search-engine
-
-[ezplatform-solr-search-engine](https://github.com/ezsystems/ezplatform-solr-search-engine)
-is the [Solr-powered](http://lucene.apache.org/solr/) search handler for eZ Platform.
-
-#### Repository Forms
-
-[Repository Forms](http://github.com/ezsystems/repository-forms) is a bundle which provides form-based interaction for the Repository Value objects.
-
-It is currently used by:
-
-- `ezsystems/platform-ui-bundle` for most management interfaces: Sections, Content Types, Roles, Policies, etc.
-- `ezsystems/ezpublish-kernel` for user registration and user generated content
-
-!!! enterprise
-
-    #### StudioUIBundle
-
-    StudioUIBundle contains the Studio editing interface
-    provided in eZ Platform Enterprise Edition.
-
-    #### Landing Page Field Type Bundle
-
-    EzLandingPageFieldTypeBundle provides
-    the Landing Page that is at the heart of StudioUI.
+The eZ Platform directory contains multiple sub-directories.
+Each sub-directory is dedicated to a specific part of the system and contains a collection of logically related files.
 
 !!! tip
 
     You can see the bundles that are automatically installed with eZ Platform in [composer.json](https://github.com/ezsystems/ezplatform/blob/master/composer.json).
+
+The following tables give an overview of the main eZ Platform directories.
+
+### Core bundles
+
+|Bundle|Description|
+|---------|-----------|
+|[ezpublish-kernel](https://github.com/ezsystems/ezpublish-kernel)|contains the core of the whole eZ Platform application e.g. EzPublishCoreBundle|
+|[repository-forms](https://github.com/ezsystems/repository-forms)|provides form-based interaction for the Repository Value objects|
+|[ezplatform-solr-search-engine](https://github.com/ezsystems/ezplatform-solr-search-engine)|[Solr-powered](http://lucene.apache.org/solr/) search handler for eZ Platform|
+|[ez-support-tools](https://github.com/ezsystems/ez-support-tools)|provides functionality for system information|
+|[ezplatform-design-engine](https://github.com/ezsystems/ezplatform-design-engine)|design fallback system for eZ Platform similar to legacy design fallback system|
+|[BehatBundle](https://github.com/ezsystems/BehatBundle)|common reusable sentence implementations and other common needs for Behat testing in eZ bundles/projects|
+
+!!! Enterprise
```

Comment:
Most of the "Enterprise" bundles are private, so people outside of our organization won't have access to that. Question is if we should link to GitHub anyway in these cases or remove the link (I would remove it personally).